src/packs/core_macros/Scan__Journal__z0pdvwE0FitMzwRJ.yml

```diff
@@ -5,106 +5,61 @@ author: jE81C9hyfb9evTbT
 img: systems/lancer/assets/icons/macro-icons/tech_quick.svg
 scope: global
 command: >-
-  //Sanity Check - Can this user even create folders and journal entries? Does
-  the folder journal folder exist?
-
+  //Sanity Check - Can this user even create folders and journal entries? Does the folder journal folder exist?
   if (JournalEntry.canUserCreate(game.user) == false) {
     ui.notifications.error(
       `${game.user.name} attempted to run SCAN to Journal but lacks proper permissions. Please correct and try again.`
     );
     return;
   }
 
-
   // ======================================================
-
   //Variables - Change these to control the macro
-
   // ======================================================
 
-
   // outputToChat - if set to true, also send the scan results to chat.
-
```

Comment:
It looks like you directly edited the yml file, so this won't actually run because all these lines get smashed into a single line. You need to edit the macro in foundry and then unpack the db with `npx gulp export_packs`

![image](https://github.com/Eranziel/foundryvtt-lancer/assets/182554/a943e2b6-6b37-4d57-90f5-cdaeb9381638)